install.py

```diff
@@ -116,7 +116,10 @@ def create_environment_json(path):
     "static": "static_root",
     "secret-key": secret_key,
     "test_visible": False,
-    "debug": True
+    "debug": True,
+    "session_security_expire_at_browser_close" : True,
```

Comment:
@alexanderward some more settings for the docker env.

Reply:
Can you add the same line to `dev_env/core/runner.py` under the `def generate_config` section?